src/Collections-Strings/String.class.st

```diff
@@ -2178,6 +2178,19 @@ String >> match: text [
 	^ self startingAt: 1 match: text startingAt: 1
 ]
 
+{ #category : #converting }
+String >> normalizeCamelCase [
+	"
+	'TheRollingStones' normalizeCamelCase >>> 'The Rolling Stones'
+	"
+	^ self class streamContents: [ :out |
+		self do: [ :e |
```

Comment:
why do you call the block parameter "e" and not "c" (like Character) or better "char" or even better "character"

Reply:
Ok, good suggestions, I just integrated them and fixed the case to not write two consecutive blank spaces where there is a single space in the input string.